package.json

```diff
@@ -56,7 +56,7 @@
     "@types/detect-browser": "^1.6.0",
     "@types/enzyme": "2.8.0",
     "@types/google.analytics": "0.0.39",
-    "@types/jquery": "2.0.46",
+    "@types/jquery": "^3.3.31",
```

Comment:
we should make sure this works with jQuery migrate.  i think it's still necessary from some legacy stuff

Reply:
im only updating the types, not the jquery version though

Reply:
nevermind.  e2e tests should catch any failures